packaging/homebrew/bee-get-addr

```diff
@@ -13,7 +13,7 @@ After you fix configuration run 'bee-get-addr' again.
         ETH_ADDRESS=$(echo "$RESP" | grep ethereum | cut -d' ' -f6 | tr -d '"')
         echo "
 Please make sure there is sufficient eth and bzz available on $ETH_ADDRESS address.
-You can get both Goerli ETH and Goerli BZZ now via the bzzaar at https://bzz.ethswarm.org/?transaction=buy&amount=10&slippage=30&receiver=0x$ETH_ADDRESS
+You can get both Goerli ETH and Goerli BZZ now via the bzzaar at https://docs.ethswarm.org/docs/installation/fund-your-node
```

Comment:
learn how to fund your node by visiting our docs at.... the same goes everywhere

Reply:
Ok, better.

Reply:
addressed

Reply:
Well, https://docs.ethswarm.org/docs/installation/fund-your-node gives me:

Page Not Found
We could not find what you were looking for.

Please contact the owner of the site that linked you to the original URL and let them know their link is broken.

Hopefully that doesn't get missed before the release!